### DISCUSSION TITLE
Directus Cloud cookie settings

### DISCUSSION THREAD

**Discussion by artecoop** [directus/directus#16769] [Archive]
Hello all!

When I try to login with the default flow, either in localhost or production deployment, I always have this error:

`Cookie “directus_refresh_token” has been rejected because it is in a cross-site context and its “SameSite” is “Lax” or “Strict”.`

Is it possible to set somewhere in the Directus Cloud project the cookie settings?

Thanks in advance!

**Comment by rijkvanzanten** [comment]
Those are not currently configurable, although that's coming soon!
